Cargo.toml

```diff
@@ -47,6 +48,7 @@ tokio = { version = "1", features = ["full"] }
 hyper = { version = "1.1.0", features = ["server"] }
 hyper-util = { version = "0.1.3", features = ["server", "http1"] }
 serial_test = "3.0"
+fantoccini = { path = "../fantoccini", features = ["test_helpers"] }
```

Comment:
This doesn't look right — how can fantoccini depend on itself with a path of `..`?

Reply:
Thanks for the feedback. I’ve removed the path reference, it was a misconfiguration introduced during testing `test_helpers` feature.